src/App.jsx

```diff
@@ -0,0 +1,12 @@
+import ProjectRouter from "./components/Router/Router"
+
+function App() {
```

Comment:
- [ ]  Please, kindly this main App functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards 👍

Reply:
Thank you so much for your feedback. This has been resolved now.

Reply:
Thank you so much for your feedback. This has been resolved now

Reply:
ES6 arrow functions used as you recommended,

![arrow -function](https://github.com/EddyShimwa/Rent-A-Ride/assets/32777408/edb5d9a6-33b9-4f9a-b06b-f30c8635861e)